openeihttp/__init__.py

```
@@ -264,18 +264,14 @@ def next_energy_rate_structure_time(self) -> datetime.datetime | None:
     def next_rate_schedule(
         self, start: datetime.datetime, rate_type: str
     ) -> tuple[datetime.datetime | None, int | None]:
-        """
-        Return the next datetime at which the rate structure changes and the new rate structure.
-
-        This function is optimzied to avoid looping over every hour, day, month combination.
-        """
+        """Return the next datetime at which the rate structure changes."""
         assert self._data is not None
         if not f"{rate_type}ratestructure" in self._data:
```

Ruff (0.14.10)

Test for membership should be `not in`

Convert to `not in`

(E713)